13-Roman_Data_Models/13-Roman_Data_Models.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<p>Add links to repositories as in <a href="https://github.com/spacetelescope/rad" target="_blank">rad</a></p><p> </p><p>Remove <strong>Note on system described below</strong> -&gt; I think the idea is for them to use rad and roman_datamodels </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Removed.</p>